lib/ansible/utils/display.py

```diff
@@ -326,6 +326,9 @@ def display(self, msg, color=None, stderr=False, screen_only=False, log_only=Fal
         Note: msg *must* be a unicode string to prevent UnicodeError tracebacks.
         """
 
+        if not isinstance(msg, str):
+            raise TypeError(f'Display message must be str, not: {msg.__class__.__name__}')
```

Comment:
FWIW, I debated making this one of our exceptions, but I figure it actually presents better for debugging if it isn't.

Otherwise we don't get the `-vvv` message, and the ability to see the line that generated the error.

Reply:
It's always a good idea to use built-in exceptions if they accurately represent the situation.